### DISCUSSION TITLE
White window with Selenium headless on Windows

### DISCUSSION THREAD

**Discussion by oculunto** [seleniumbase/SeleniumBase#3141] [Q&A] [answered]
Hello, 
I just updated Chrome and Seleniumbase to the latest versions, and I am having a problem with headless mode on Windows 10 and 11. Starting chrome with headless ON spawns an uninteractable white window, everything else works fine, but that window stays there and you cannot even close it manually (it closes when closing the driver, or killing all chrome tasks). 
Here is someone having the same issue: 
`https://stackoverflow.com/questions/78996364/chrome-129-headless-shows-blank-window` 
Does anyone have this too ?

**Comment by mdmintz** [comment, marked as answer]
This was resolved in `4.31.0` - https://github.com/seleniumbase/SeleniumBase/releases/tag/v4.31.0
